media/changes.xml

```diff
@@ -30,6 +30,9 @@
       <action type="add" dev="mrozati" issue="WHAN-33">
         ResourceMedia: Add support for request attributes defining responsive image sizes. This can be either `imageSizes` and `widthOptions` (width + required flag), or `pictureSourceMediaFormat`, `pictureSourceMedia` and `pictureSourceWidths`.
       </action>
+      <action type="fix" dev="mrozati" issue="WHAN-34">
```

Comment:
should probably be "update", the ticket is also labeled as improvement.

Reply:
done